Formula/cargo.rb

```diff
@@ -0,0 +1,27 @@
+require File.expand_path("../../Homebrew/emacs_formula", __FILE__)
+
+class Cargo < EmacsFormula
+  desc "Emacs Minor Mode for Cargo, Rust's Package Manager."
+  homepage "https://github.com/kwrooijen/cargo.el"
+  url "https://melpa.org/packages/cargo-20160525.519.tar"
```

Comment:
Right, luckily `cargo` has a release on GitHub. I've updated it to use that instead.